src/lib/containers/SearchV2.tsx

```diff
@@ -0,0 +1,227 @@
+import * as React from 'react'
+import { QueryWrapperChildProps } from './QueryWrapper'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import {
+  faCaretDown,
+  faCaretUp,
+  faSearch,
+  faTimes,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { insertConditionsFromSearchParams } from '../utils/functions/sqlFunctions'
+import { unCamelCase } from '../utils/functions/unCamelCase'
+
+library.add(faCaretDown)
+library.add(faCaretUp)
+library.add(faSearch)
+library.add(faTimes)
+
+type SearchState = {
+  show: boolean
+  searchText: string
+  columnName: string
+}
+
+export type SearchProps = {
+  isQueryWrapperMenuChild?: boolean
+}
+
+type InternalSearchProps = QueryWrapperChildProps & SearchProps
+
+/*
+  Do we want to encode the like clause in the URL?
```

Comment:
No, let's wait for the Conditions API (then this will be a part of the query request and I think will automatically be part of the URL param, then we can use the Conditions and Selected Facets to create the _chicklets_).